sources/europe/de/Erlangen2020.geojson

```diff
@@ -0,0 +1,211 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "Erlangen-2020",
+        "name": "Erlangen TrueDOP (2020 5,0 cm)",
+        "type": "tms",
+        "category": "photo",
+        "start_date": "2020-03-28",
+        "end_date": "2020-03-28",
+        "url": "https://geodaten.erlangen.de/luftbilder/2020/{zoom}/{x}/{y}.png",
+        "max_zoom": 21,
+        "license_url": "https://geodaten.erlangen.de/luftbilder/2020/license/index.html",
```

Comment:
I'm unable to access this URL, is it accessible for you?

Reply:
sorry, we had some issues tonight, but could fix them now. The ressources should be available again